https:/github.com/SiegeEngineers/aoc-reference-data.git

```diff
@@ -0,0 +1,9 @@
+ name: Good_Game
+  country: Russia
+  aoeelo: 87
+  liquipedia: Good_Game
+  twitch: https://www.twitch.tv/good_game_aoe
+  de:
+   1165138'   
+  # henry
+  team: [Rus-F]
```

Comment:
```suggestion
  team: SalzZ
```

Reply:
See liquipedia transfer
![2021-05-19 02_13_59-Liquipedia Age of Empires Wiki — Mozilla Firefox-Screenshot](https://user-images.githubusercontent.com/14062932/118738692-e939cc00-b847-11eb-86cf-70aa26aca9b3.png)